CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Changelog for Critical Maps iOS
 - Fix missing information for VoiceOver user for the navigationbar buttons
 - Fix VoiceControl labels on each view
 - Fix `bottomContentOffset` for NavigationOverlay (did hide the Legal button)
+- Fix `BikeAnnotation` it is more accesible now(scales with preferredContentSizeCategory)
```

Comment:
Sorry for that, I added a new `Unreleased` section on top of the changelog. Could you please merge the latest change from  `master` and move your line on top to the unreleased section? :)

Reply:
Just a minor suggestion:

```suggestion
- Make `BikeAnnotation` more accessible (scales with preferredContentSizeCategory)
```

Reply:
I agree with you, just made the changes